README.md

```diff
@@ -34,6 +36,15 @@ bucket.company("acme_inc", { name: "Acme Inc", plan: "pro" }, "john_doe");
 
 // track events
 bucket.track("sent_message", { foo: "bar" }, "john_doe");
+
+// collect qualitative feedback
+bucket.feedback({
+  featureId: "my_feature_id",
+  userId: "john_doe",
+  companyId: "Acme Inc", // String (optional)
```

Comment:
let's keep `userId` and `companyId` in the same style. So: `acme_inc` for example

Reply:
👍